### PR TITLE
Don't create new animationFrame on every frame

### PR DESCRIPTION
**Pull request by gsathya**

### CODE REVIEW

**Comment by gsathya** [comment]
I don't think I need to sign the CLA. gunasekaran@google.com is me

File: demo/index.html [hunk 1]
```diff
@@ -99,7 +99,7 @@
         }
         progress.value = progress.min;
       }
-      var animationFrame = requestAnimationFrame(nextProgress);
```
Comment (keanulee) [inline demo/index.html:102]: @gsathya I don't even know why we keep a reference to `animationFrame` at all. Mind removing all references to `animationFrame`?

**Comment by keanulee** [comment]
@gsathya You should just sign the CLA (with your personal email/whatever your git config has) anyways. Not really needed for compliance in your case, but just to make @googlebot stop complaining :)

**Comment by keanulee** [comment]
LGTM. Thanks.

**Pull request merged**